src/utils/fileUtils.js

```diff
@@ -118,6 +118,10 @@ const genFileInfo = function(obj) {
 					: data
 			}
 		}
+
+		if (key === 'filename') {
+			fileInfo[camelcase(key)] = escape(data)
```

Comment:
We should avoid touching the fileinfos. They should stay as pristine as what the dav backend is giving us :)
I would rather have this fixed in  the vue router side: https://github.com/nextcloud/photos/blob/7ed958ced085f4548dafe7646d26e801e39732e4/src/router/index.js#L57

Reply:
I'll try

Reply:
Can't find a way to do this..

Reply:
Let's talk tomorrow :)
You're here already?

Reply:
I'll be there on Friday :/

Reply:
Oh, you're only there for two days?

Reply:
Yes, I'll explain you why, but we're underwater at work

Reply:
Arf :(
I'll figure it out :)